Templates/Swift/Sources/APIService.swift

```diff
@@ -8,16 +8,16 @@ public struct APIService<ResponseType: APIResponseValue> {
     public let path: String
     public let hasBody: Bool
     public let isUpload: Bool
-    public let securityRequirement: SecurityRequirement?
+    public let securityRequirements: [SecurityRequirement]?
 
-    public init(id: String, tag: String = "", method:String, path:String, hasBody: Bool, isUpload: Bool = false, securityRequirement: SecurityRequirement? = nil) {
+    public init(id: String, tag: String = "", method:String, path:String, hasBody: Bool, isUpload: Bool = false, securityRequirements: [SecurityRequirement]? = nil) {
```

Comment:
Maybe we could make this a non optional that defaults to `[]`?

Reply:
Sure, I am going to supply this change immediately.